.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
```

Comment:
- [ ]  [REQUIRED] You did well adding this text.md file name to gitignore. But according to this project requirements, you are suppose to add a text.md file with the text "This file should be ignored by git". Kindly implement this.